spec/std/time/time_spec.cr

```diff
@@ -363,8 +363,8 @@ describe Time do
       t2 = t.shift months: 1
       t2.should eq Time.utc(2014, 11, 30, 21, 18, 13)
 
-      t2 = t.shift months: 1
-      t2.should eq Time.utc(2014, 11, 30, 21, 18, 13)
+      t2 = t.shift months: -1
```

Comment:
Here there are same code twice. I guess that the former wants to test `plus` and the latter wants to test` minus`.